generators/language-go/index.js

```diff
@@ -0,0 +1,177 @@
+/*
+ * © Copyright IBM Corp. 2017, 2018
```

Comment:
for a brand-new file, you only need to use `2018`. Please delete the `2017`

Reply:
done!